cmd/vsphere-xcopy-volume-populator/internal/populator/secure-vmkfstools-wrapper.py

```
@@ -0,0 +1,292 @@
+        logging.info("Source VMDK path validation passed")
+        
+        logging.info("Validating target LUN path...")
+        target_lun = validate_path(target_lun)
+        logging.info("Target LUN path validation passed")
+        
+        task_id = str(uuid.uuid4())
+        tmp_dir = TMP_PREFIX.format(task_id)
+        os.makedirs(tmp_dir, mode=0o750, exist_ok=True)
+        rdmfile = f"{source_vmdk}-rdmdisk-{os.getpid()}"
+
+        stdout_file = open(os.path.join(tmp_dir, "out"), "w")
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)